README.md

```diff
@@ -197,94 +182,137 @@ coastal area.
 
-This scenario simulates a system that consists of a cubic box filled with water molecules, evolving according to the rules of Molecular Dynamics: this means that the water molecules' positions are updated according to Newton's equation in discrete time steps. The force that acts upon the particles is computed using standard molecular force fields. The implementation of this scenario was inspired by [this article](https://arxiv.org/abs/2112.03383). 
+### Simulators Available
 
-#### Example
+The simulators we provide are all open-source and have their own dedicated documentation.
 
-First we initialize the scenario:
-```
-from inductiva import molecules
+Currently, we have available the following simulators:
+- [SPlisHSPlasH](https://github.com/InteractiveComputerGraphics/SPlisHSPlasH)
+- [DualSPHysics](https://github.com/DualSPHysics/DualSPHysics)
+- [OpenFOAM](https://www.openfoam.com/)
+- [SWASH](https://swash.sourceforge.io/)
+- [xBeach](https://oss.deltares.nl/web/xbeach/)
+- [GROMACS](https://www.gromacs.org/)
 
-scenario = molecules.scenarios.MDWaterBox(temperature = 300, box_size = 2.3)
-```
+If you would like other simulators to be added, contact us at [simulations@inductiva.ai](mailto:simulations@inductiva.ai).
 
-The user can specify the temperature (in Kelvin) and the box size (length of one of the cube's edges, in nanometers). The numbers above correspond to the default values.
+### Example
 
-After the initialization, we are ready to simulate the system:
+Example of how to use the simulators:
 
 ```python
-output = scenario.simulate(simulation_time = 10, nsteps_minim = 5000)
-```
-output = scenario.simulate(simulation_time = 10,
-                           nsteps_minim = 5000)
+from inductiva import fluids
 
-The simulate method initializes a simulation using cloud resources. The user can set the duration of the simulation in nanoseconds and the number of steps for the energy minimization.
+simulator = fluids.simulators.DualSPHysics()
 
-Finally, we can visualize the simulation: 
-```python
-view = output.render_interactive()
-view
+output_dir = simulator.run(input_dir="FlowCylinder",
+                           sim_config_filename="CaseFlowCylinder_Re200_Def.xml",
+                           output_dir="Flow",
+                           device="gpu")
 ```
-This renders the trajectory in an interactive visualization that can be manipulated in a jupyter notebook: 
-<p align="center">
-  <img width="900" height="300" src="https://github.com/inductiva/inductiva/assets/114397668/6cc809ed-085b-4fc6-b1a6-358be03cd061">
-</p>
 
+The user must specify the input directory, the simulation configuration file, the output directory and the device to run the simulation on.
 
-### ProteinSolvation
+Visualize the results:
```

Comment:
Isn't there missing a command line for the visualization?

Reply:
Or remove this visualization, since we don't provide visualization utilities for arbitrary data generated from the simulators?

Reply:
There is none ready, maybe I should omit this part...